_layouts/homepage.html

```diff
@@ -70,8 +70,16 @@ <h1>{{ page.secondary.heading }}</h1>
       </a>
       {% endif %}
       <a href="{{ page.ctas.roadmap.url }}" class="link-readmore" target="_blank">{{ page.ctas.roadmap.text }}</a>
-      {% assign latest = site.versions | sort: 'version_sort' | reverse | first %}
-      <div class="latest-version">{{ page.version_feature.latest_label}} {{ latest.version }} {{ page.version_feature.date_label}} {{ latest.date | date_to_string: "ordinal", "US"}}</div>
+      {% assign versions = site.versions | sort: 'version_sort' | reverse %}
+      {% assign latest = versions | first %}
+      {% for version in versions %}
+          {% assign major_version = version.version | split: "." | first %}
+          {% if major_version == 1 %}
```

Comment:
Okay so the reason it is not rendering comes down to this. Here you are comparing the string's '1', '2' to the number 1. We could fix this to be '1' but that leaves us with the issue of updating this every time we have a major release which I am not a fan of so I have 2 other recommendations outlined in the above sections.

Reply:
3. Finally, we can now compare the current major from the iteration to the previous major. 

```{% if major_version == previous_major %}```